templates/main.php

```diff
@@ -62,7 +62,8 @@
 		<?php print_unescaped($this->inc('part.navigation')); ?>
 		<?php /* print_unescaped($this->inc('part.settings')); */ ?>
 	</div>
-	<div id="app-content" ng-class="{ 'details-visible': sidebar.show }" ui-view>
+	<div id="app-content" ng-class="{ 'details-visible': sidebar.show }">
+		<div ui-view></div>
```

Comment:
@pixelipo This is part of the logic from core, so that apps don't need to care about adding this sidebar slide functionality. But since we use our own js code to toggle those of course we could also  just keep the existing structure and add the toggle button to the template.